distro/run/assembly/resources/default.yml

```diff
@@ -15,6 +15,7 @@ camunda.bpm:
       allowed-origins: "*"
     rest:
       disable-wadl: false
+    deploy-changed-only: true
```

Comment:
Name Selection Explanation:

Didn't follow convention of `deploy-changed-only.enabled` or `cors.enabled` due to:
- `auth` and `cors ` have other properties as well and `enabled` is one of them
- `deploy-changed-only` is standalone